docs/docs/guides/web3_config/index.md

```diff
@@ -201,44 +202,394 @@ console.log(web3.getContextObject().config)
-    number: FMT_NUMBER.BIGINT,
-    bytes: FMT_BYTES.HEX,
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.handleRevert = false;
+
+console.log(web3.getContextObject().config);
+```
+
+### [defaultAccount](/api/web3-core/class/Web3Config#defaultAccount)
+This `defaultAccount` is used as the default `from` property, if no `from` property is specified in for the following methods:
+- [`web3.eth.sendTransaction()`](/api/web3-eth/function/sendTransaction);
+- [`web3.eth.call()`](/api/web3-eth/function/call);
+- [`myContract.methods.myMethod().call()`](/libdocs/Contract#call);
+- [`myContract.methods.myMethod().send()`](/libdocs/Contract#send);
+
+The default value of `defaultAccount` is `undefined`.
+
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.defaultAccount = undefined;
+
+console.log(web3.getContextObject().config);
+```
+
+### [defaultBlock](/api/web3-core/class/Web3Config#defaultBlock)
+The following methods require a `blockNumber` parameter during its execution process, the `defaultBlock` is used for these methods. 
+- [`web3.eth.getBalance()`](/api/web3-eth/function/getBalance);
+- [`web3.eth.getCode()`](/api/web3-eth/function/getCode);
+- [`web3.eth.getTransactionCount()`](/api/web3-eth/function/getTransactionCount);
+- [`web3.eth.getStorageAt()`](/api/web3-eth/function/getStorageAt);
+- [`web3.eth.call()`](/api/web3-eth/function/call);
+- [`myContract.methods.myMethod().call()`](/libdocs/Contract#call);
+
+You can override it by passing in the defaultBlock as last parameter.
+
+The default value of `defaultBlock` is "latest".
+
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.defaultBlock = "latest";
+
+console.log(web3.getContextObject().config);
+```
+
+#### All available choices for defaultBlock:
+```ts
+web3.defaultBlock = 20167235; // A block number
+web3.defaultBlock = "earliest"; // The genesis block
+web3.defaultBlock = "latest"; // The latest block (current head of the blockchain)
+web3.defaultBlock = "pending"; // The currently mined block (including pending transactions)
+web3.defaultBlock = "finalized"; // (For POS networks) The finalized block is one which has been accepted as canonical by greater than 2/3 of validators
+web3.defaultBlock = "safe"; // (For POS networks) The safe head block is one which under normal network conditions, is expected to be included in the canonical chain. Under normal network conditions the safe head and the actual tip of the chain will be equivalent (with safe head trailing only by a few seconds). Safe heads will be less likely to be reorged than the proof of work network`s latest blocks.
+```
+
+### [transactionBlockTimeout](/api/web3-core/class/Web3Config#transactionBlockTimeout)
+The `transactionBlockTimeout` is used over socket-based connections. This option defines the number of new blocks to wait for the **first confirmation**, otherwise the PromiEvent rejects with a timeout error.
```

Comment:
```suggestion
 This option defines the number of new blocks to wait for the **first confirmation**, otherwise the PromiEvent rejects with a timeout error.
```
Its there with polling as well,

Reply:
I removed all socket-based connections and HTTP connections from the doc.